packages/teleport-generator-core/src/validator/index.ts

```diff
@@ -1,6 +1,6 @@
 import Ajv from 'ajv'
-import componentSchema from '@teleporthq/teleport-uidl-definitions/lib/schemas/component.json'
-import projectSchema from '@teleporthq/teleport-uidl-definitions/lib/schemas/project.json'
+import componentSchema from '@teleporthq/teleport-generator-shared/src/uidl-definitions/schemas/component.json'
+import projectSchema from '@teleporthq/teleport-generator-shared/src/uidl-definitions/schemas/project.json'
```

Comment:
Hey @alexnm can we adopt the step which I used in the other branch here --> https://github.com/teleporthq/teleport-code-generators/pull/194/files#diff-fa69e88ee0eea386a078ae0237eff9f7R21, since in all the other packages we import things under ``/lib/`` and only in this specific case we directly import from ``src`` to eliminate any confusions while using in future i just wrote a simple copy command to create and expose a ``lib`` instead of ```src`` 😅

Reply:
this is just a temporary thing, I wasn't sure where to keep the schemas and the mappings, probably will move them closer to where they are used so there's no need to copy them to the output lib folder. will update on this one